docs/debugging.md

```diff
@@ -0,0 +1,117 @@
+# Debugging your scripts locally
+
+For a complete example, see the [debugging example](../examples/debugging/README.md).
+
+- [Debugging your scripts locally](#debugging-your-scripts-locally)
+  - [Dependencies](#dependencies)
+  - [Run your code locally](#run-your-code-locally)
+    - [Use assets from a deployed Substra platform and local assets](#use-assets-from-a-deployed-substra-platform-and-local-assets)
+    - [Dependency between assets](#dependency-between-assets)
+    - [Debugging with logs and breakpoints](#debugging-with-logs-and-breakpoints)
+      - [Access the logs](#access-the-logs)
+      - [Debug with pdb](#debug-with-pdb)
+
+## Dependencies
+
+The required dependencies are:
+
+- Docker:
+  - Mac: [Docker Desktop](https://www.docker.com/products/docker-desktop)
+  - Ubuntu/Debian: `sudo apt install docker docker-compose`
+- Python 3 (recommended 3.6 or 3.7)
+  - It is recommended to use a virtual environment to install Substra packages (for instance [virtualenv](https://virtualenv.pypa.io/en/latest/))
+- Substra package
```

Comment:
This is already a requirement for running substra so I would keep only stuff specific to the debugging mode.

Reply:
You are right, I only left that Substra must be installed and the Docker requirement.